qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/RunTest.java

```diff
@@ -31,6 +31,7 @@
 		public static final String PERSON = "@person";
 		public static final String ALLERGIES = "@allergies";
 		public static final String INITIAL_SETUP = "@initialSetup";
+		public static final String XSS_VULNERABILITY = "@xssVulnerability";
```

Comment:
Hello @kdaud  Looking forward for reviews, recommendations if everything looks fine. Merge. thanks